apps/dictionary/scripts/build-config/utils.js

```diff
@@ -136,12 +124,27 @@ StyleDictionaryPackage.registerTransform({
 /**
  * @typedef {Object} CssBuildConfig - Configuration for building CSS from Token Studio design tokens.
  * @property {string[]} sources - The design token files to include.
- * @property {string[]|undefined} includes - The component design token files to include.
+ * @property {string[]|undefined} [includes] - The component design token files to include. Include tokens will be overridden by "source" attribute tokens. This should be used for sub-brands that reference their parent brand.
  * @property {string} destination - The output file path.
  * @property {TokenFilter|undefined} [tokenFilter] - A function to filter tokens to include.
  * @property {string|undefined} [parentSelector] - A parent CSS selector for generated CSS.
  */
 
+const transformers = [
+	'Origami/pxToRem',
+	'ts/size/px',
+	'ts/size/lineheight',
+	'ts/descriptionToComment',
+	'ts/typography/css/shorthand',
+	'ts/typography/css/fontFamily',
+	'ts/border/css/shorthand',
+	'ts/shadow/css/shorthand',
+	'ts/color/css/hexrgba',
+	'ts/color/modifiers',
+	'name/cti/kebab',
+	'name/origamiPrivatePrefix',
+];
+
```

Comment:
This breaks the flow of the two JSDoc comments, reordering will keep the two JSDoc comments nearby for relevance.

```suggestion
const transformers = [
	'Origami/pxToRem',
	'ts/size/px',
	'ts/size/lineheight',
	'ts/descriptionToComment',
	'ts/typography/css/shorthand',
	'ts/typography/css/fontFamily',
	'ts/border/css/shorthand',
	'ts/shadow/css/shorthand',
	'ts/color/css/hexrgba',
	'ts/color/modifiers',
	'name/cti/kebab',
	'name/origamiPrivatePrefix',
];

 * @property {string} destination - The output file path.
 * @property {TokenFilter|undefined} [tokenFilter] - A function to filter tokens to include.
 * @property {string|undefined} [parentSelector] - A parent CSS selector for generated CSS.
 */
```